src/debug_menu.cpp

```diff
@@ -2117,6 +2119,12 @@ void debug()
         }
         break;
 
+        case DEBUG_JOIN_DISCORD: {
+            open_url( "https://discord.gg/mXUSMGTc" );
```

Comment:
```suggestion
            open_url( "https://discord.gg/XW7XhXuZ89" );
```

could we use the permanent link in `#information`?

![image](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/54838975/4aedada5-c470-4170-b682-58f7eb42eb9d)